cpp/CMakeLists.txt

```diff
@@ -105,6 +105,11 @@ find_package(Threads REQUIRED)
 find_package(JNI REQUIRED)
 
 if(BUILD_TESTS)
+  set(GLUTEN_GTEST_MIN_VERSION "1.12.1")
```

Comment:
Maybe 1.13.0 is more meaningful. We should keep gluten gtest version as same as velox.

Reply:
Thanks, already fix it.